coalib/bearlib/languages/definitions/Shell.py

```diff
@@ -14,5 +12,6 @@ class Shell:
     keywords = [
         'if', 'then', 'else', 'elif', 'fi', 'case', 'coproc', 'esac',
         'for', 'select', 'while', 'until', 'do', 'done', 'in',
-        'function']
-    special_chars = list('{}!|><[]')
+        'function',
```

Comment:
do not modify this line.

when you write new lines, or modify existing lines, do you very best.

when you dont need to touch other lines, do not touch them.